source/_integrations/zha.markdown

```diff
@@ -380,27 +382,36 @@ This service disables a lock code on a Zigbee lock.
+
+**To add a new Zigbee device:**
 
 1. Go to the **Integrations** panel, find the **Zigbee Home Automation** integration that was added by the configuration steps above, and select **Configure**.
 1. Click on the plus button at the bottom right corner to start a scan for new devices.
-1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
+1. Reset your Zigbee devices to fectory default settings according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
-### Using router devices
+### Using router devices to add more devices
+
+You can normally assume that most mains-powered devices, e.g., many always-powered wall plugs or lightbulbs in your Zigbee network will automatically act as a Zigbee Router Device (sometimes also referred to as a Zigbee "signal repeater" or "range extended"). 
+
+Because Zigbee should use a "[wireless mesh network]"(https://en.wikipedia.org/wiki/Wireless_mesh_network) to be effective you will need to add Zigbee Router devices to increase the number of Zigbee devices that can be used in your Zigbee network, both in the total number of devices that can be added as well as the total range and coverage of the network. You should also be aware that some Zigbee Router devices do a much better job at routing/repeating Zigbee signals/messages than some other devices, and you should not have a setup where Zigbee Router devices (e.g. lightbulbs) are often powered-off as Zigbee Router devices are meant to be always available.
+
+All Zigbee Coordinator firmware will only allow you to directly connect a certain amount of devices, and that limit is set for two reasons; firstly to not overload the Zigbee Coordinator SoC, and secondly to encourage your Zigbee network to quickly begin to utilize a "[mesh networking](https://en.wikipedia.org/wiki/Mesh_networking)" topology instead of only a "[star network](https://en.wikipedia.org/wiki/Star_network)" topology.
```

Comment:
```suggestion
All Zigbee coordinator firmware will only allow you to directly connect a certain amount of devices. That limit is set for two reasons; firstly, to not overload the Zigbee coordinator, and secondly, to encourage your Zigbee network to quickly begin to utilize a "[mesh networking](https://en.wikipedia.org/wiki/Mesh_networking)" topology instead of only a "[star network](https://en.wikipedia.org/wiki/Star_network)" topology.
```
Removed the system on chip. I think there is already enough technical terminology in there, and I'm not sure that knowing that it's the SoC that is being overloaded makes much of a difference to the user.